corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/list/list_controller.js

```diff
@@ -0,0 +1,18 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("AppSelect.AppList", function(AppList, FormplayerFrontend, Backbone, Marionette, $){
+    AppList.Controller = {
+        listApps: function(){
+            var fetchingApps = FormplayerFrontend.request("appselect:apps");
```

Comment:
shouldn't the pattern be `collection new AppCollection(); collection.fetch().done...`?